cmd/govim/testdata/scenario_default/quickfix_replaced_dep.txt

```diff
@@ -0,0 +1,63 @@
+# Test that we get diagnostics for edits in a locally replaced dependency
```

Comment:
Perhaps:

```
Test that we get diagnostics for errors in directory-replaced module requirements
```

The use of the word "directory" is a bit more specific and consistent with the way such module replaces are described.

Reply:
Fixed